### PR TITLE
Project create modal

### PR DESCRIPTION
**Pull request by Civolilah**
Here is the screenshot of implemented solution for adding projects using modal:

Screenshot 2023-01-22 at 15 48 05

### CODE REVIEW

File: src/pages/projects/common/components/CreateProjectModal.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    >
+      {project ? (
+        <>
+          <Element leftSide={t('project_name')}>
+            <InputField
+              value={project.name}
+              onValueChange={(value) => handleChange('name', value)}
+              errorMessage={errors?.errors.name}
+            />
+          </Element>
```
Comment (beganovich) [inline src/pages/projects/common/components/CreateProjectModal.tsx:141]: `<Element>`  is meant for usage in cards, not modals. This way we end up with double padding, causing elements to not align.

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, just drop the element and use the modal's native wrapping.

File: src/pages/projects/common/components/CreateProjectModal.tsx (hunk 1 above)
Comment (Civolilah) [reply src/pages/projects/common/components/CreateProjectModal.tsx:141]: In that case, I removed the `Element` component and the name of the filed is managed with the `label` and `inputLabel` props. But the whole UI doesn't look good with only one column IMO. So, I separated it into two columns of 4 fields. The description has been updated with a screenshot of the new UI. Let me know your thoughts.

**Comment by beganovich** [comment]
@turbo124 thoughts?

**Comment by turbo124** [comment]
@beganovich LGTM!

**Review by beganovich — APPROVED** [review]
Looks good, thanks 👍

**Pull request merged**